src/components/FreestarAdSlot/freestarWrapper.js

```diff
@@ -226,7 +226,7 @@ class FreestarWrapper {
   }
 
   newAdSlot (placementName, onNewAdSlotsHook, channel, targeting, adUnitPath, slotSize, sizeMappings) {
-    if (this.queueAdCalls) {
+    if (this.queue) {
```

Comment:
`queueAdCalls` is the method used to toggle the state of `this.queue` and does not have a return value